server/database/seed.js

```diff
@@ -30,7 +30,7 @@ const seed = (storeStart, storeEnd) => {
 // START = 50,000
 // END = 60,000
 var storeCountStart = 0
-var storeCountEnd = 1000
+var storeCountEnd = 100
 var start = StopWatch.start('seconds');
```

Comment:
these variables should probably be constant

Reply:
That's something I've been trying to pay attention to and glad you commented that, changes should reflect on next commit. Although, those two actually increment so that the store id increments for every step so in this case I'll need to keep it var.